tools/codegen/gen_backend_stubs.py

```diff
@@ -295,7 +316,7 @@ def make_file_manager(install_dir: str) -> FileManager:
         class_name = backend_indices[backend_dispatch_key].native_function_class_name()
 
         if impl_path is not None:
-            error_on_missing_kernels(native_functions, backend_indices, backend_key, autograd_key, impl_path)
+            error_on_missing_or_invalid_kernels(native_functions, backend_indices, backend_key, autograd_key, impl_path)
```

Comment:
for invalid kernel checking, it should be done no matter impl_path is provided or not.

we may move 'if impl_path is not None:' into function error_on_missing_or_invalid_kernels.

Reply:
fair point. I'm going to re-make this PR and base it against master after #69020 lands, but I'll make that change when I do.